helm/stunner-gateway-operator/templates/stunner-gateway-operator.yaml

```diff
@@ -382,8 +383,7 @@ spec:
         - containerPort: 13478
           name: cds
           protocol: TCP
-        securityContext:
-          allowPrivilegeEscalation: false
+        securityContext: {{- toYaml .Values.stunnerGatewayOperator.deployment.container.manager.securityContext | nindent 10 }}
       securityContext:
         runAsNonRoot: true
```

Comment:
@cjvirtucio87 Do you want to only manage the container's securityContext, or this one also?

Reply:
the PSA controller only cares about the container-level `securityContext`, but I guess it can't hurt. fixed: 56c62a5804c6718d63a45888299f22b1203cd7dc

```yaml
---
stunnerGatewayOperator:
  deployment:
    securityContext:
      runAsUser: 1000
      runAsNonRoot: true
```

```console
% kubectl --namespace stunner get deployments/stunner-gateway-operator-controller-manager --output json | jq .spec.template.spec.securityContext
{
  "runAsNonRoot": true,
  "runAsUser": 1000
}
```

Reply:
thank you